src/storage/src/hummock/sstable_store.rs

```diff
@@ -354,20 +323,25 @@ impl SstableStore {
 
     pub async fn sstable(
         &self,
-        sst_id: HummockSstableId,
+        sst: &SstableInfo,
         stats: &mut StoreLocalStatistic,
     ) -> HummockResult<TableHolder> {
         stats.cache_meta_block_total += 1;
+        let sst_id = sst.id;
         self.meta_cache
             .lookup_with_request_dedup::<_, HummockError, _>(sst_id, sst_id, || {
                 let store = self.store.clone();
-                let meta_path = self.get_sst_meta_path(sst_id);
+                let meta_path = self.get_sst_data_path(sst_id);
```

Comment:
`meta_path` and `data_path` is a single object. 

Maybe we can use `get_sst_object_path` to replace `get_sst_data_path`, avoid misunderstanding.

Reply:
No. Data could also represent both data and meta....